<?xml version="1.0"?>
<launch>
<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf" output="screen">
    <param name="freq" value="100.0"/>
    <param name="sensor_timeout" value="1.0"/>
    <param name="publish_tf" value="true"/>
    <param name="odom_used" value="false"/>
    <param name="imu_used" value="true"/>
    <param name="vo_used" value="false"/>
    <remap from="imu_data" to="/os_cloud_node/imu"/>
    <param name="output_frame" value="odom"/>
    <param name="base_footprint_frame" value="os_imu"/>
</node>
</launch>
